hubspot/task_helpers.py

```diff
@@ -36,19 +39,10 @@ def sync_hubspot_deal_from_order(order):
     Args:
         order (Order): The order to sync
     """
-
-    if not isinstance(order, Order):
-        # Some tests cause order to be a string.
-        return
-
     try:
-        personal_price = PersonalPrice.objects.get(
-            user=order.user,
-            bootcamp_run=order.get_bootcamp_run()  # Under tha assumption that its one klass per order
-        )
-        sync_hubspot_deal(personal_price)
-    except PersonalPrice.DoesNotExist:
-        pass
+        sync_hubspot_deal(order.application)
+    except AttributeError:
+        log.error("No matching BootcampApplication found for order %s", order.id)
```

Comment:
Does it make sense to use `log.exception` here instead? Does the `AttributeError` contain useful information?

Reply:
I don't think the error contains useful info, this happens if the `Order` doesn't have an associated `BootcampApplication` (tries to look up `order.application.id` => `None.id`)